cmd/deploy/proxy.go

```diff
@@ -355,7 +356,8 @@ func (ph *proxyHandler) translateDeploymentSpec(body map[string]json.RawMessage)
 func (ph *proxyHandler) translateStatefulSetSpec(body map[string]json.RawMessage) error {
 	var spec appsv1.StatefulSetSpec
 	if err := json.Unmarshal(body["spec"], &spec); err != nil {
-		return fmt.Errorf("could not process statefulset spec: %s", err)
+		oktetoLog.Info(err)
```

Comment:
In that case it would be stricter and would avoid sending a request to the cluster since we are not in the case of having an invalid resource with which later the cluster will give error. Since I don't know in which cases we can have this error, I would continue with the current logic and only resend request to the cluster when we "know" that the cluster will return an error.

Reply:
True. When unmarshalling metadata I think we are in the same case. Would you include to forward request if there is an unmarshalling json.RawMessage error?

Reply:
Not sure about that one, but probably make sense. I prefer that users have to do some manual deleting than preventing from deploying their app

@pchico83 @jmacelroy WDYT?